interface/src/app/signature/signature.spec.js

```diff
@@ -21,11 +21,15 @@ describe('<high-weight-genes> directive', function() {
 
   it('should render HTML correctly', function() {
     var parentScope = $rootScope.$new();
-    parentScope.signatureID = 123;
-    var testHTML = '<high-weight-genes signature-id="{{signatureID}}" ' +
-        'genes="ctrl.genes"></high-weight-genes>';
-    var uri = '/api/v0/participation?limit=0&node=' + parentScope.signatureID;
+    parentScope.signatureId = 123;
+    parentScope.selectedParticipationType = null;
+
+    var testHTML = '<participatory-genes signature-id="{{signatureId}}" ' +
+        'genes="genes" selected-participation-type=' +
+        '"selectedParticipationType"></participatory-genes>';
```

Comment:
I might wrap this to be a little more readable as follows:

```javascript
    var testHTML = '<participatory-genes ' +
      'signature-id="{{signatureId}}" ' +
      'genes="genes" ' +
      'selected-participation-type="selectedParticipationType">' + 
    '</participatory-genes>';
```

At any rate, that's how I'd wrap it in an HTML template. But this is a minor quibble.